### PR TITLE
Create config.json

### PR DESCRIPTION
**Pull request by jason-fox**
## Proposed changes

The FIWARE Foundation need a simple JSON file available in a standard location in each repo to be able to continue to make FIWARE Releases and improve the degree of cross-product integration testing that can occur. This change has been agreed by the TSC and has been added to the contribution requirements

## Types of changes

What types of changes does your code introduce to the project: _Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before
merging your code._

-   [x] I have read the [CONTRIBUTING](https://github.com/FIWARE/context.Orion-LD/blob/master/CONTRIBUTING.md) doc
-   [x] I have signed the [CLA](https://fiware.github.io/contribution-requirements/individual-cla.pdf)
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jason-fox** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/fiware/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "enabler": "Orion-LD Context Broker",
+    "chapter": "core",
+    "academy": "/core/orion-ld",
+    "readthedocs": "fiware-orion-ld",
+    "helpdesk": "Orion-LD",
+    "coveralls": "",
```
Comment (wistefan) [inline .github/fiware/config.json]: Available: https://coveralls.io/github/FIWARE/context.Orion-LD

File: .github/fiware/config.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "dockerregistry": ["hub.docker.com"],
+    "docker": ["fiware/orion-ld"],
+    "email": "ken.zangelin@fiware.org",
+    "status": "incubating",
+    "compose": "https://raw.githubusercontent.com/FIWARE/context.Orion-LD/develop/docker/docker-compose.yml",
+    "exclude": ["mongo"],
+    "stackexchange": ["http://stackoverflow.com/questions/tagged/fiware-orion"],
+    "unit-test": "",
+    "smoke-test": "",
+    "dockerfile": ""
```
Comment (wistefan) [inline .github/fiware/config.json]: Also available: https://github.com/FIWARE/context.Orion-LD/blob/develop/docker/Dockerfile

Comment (jason-fox) [reply .github/fiware/config.json]: Fixed  9ed1a09

File: .github/fiware/config.json (hunk 1 above)
Comment (jason-fox) [reply .github/fiware/config.json]: Fixed  9ed1a09

**Review by wistefan — APPROVED** [review]

**Pull request merged**